docs/plugins/repository-azure.asciidoc

```diff
@@ -82,13 +85,15 @@ cloud:
                 account: your_azure_storage_account1
                 key: your_azure_storage_key1
                 default: true
+                max_retries: 7
             my_account2:
                 account: your_azure_storage_account2
                 key: your_azure_storage_key2
                 timeout: 30s
 ----
 
-In this example, timeout will be 10s for `my_account1` and 30s for `my_account2`.
+In this example, timeout will be 10s for `my_account1` with 7 retries before failing
```

Comment:
Can you clarify the semantics of "10s with 7 retries"? What happens after 10 seconds (assuming we did not retry 7 times yet)? Does it fail? My assumption is that it gives up either after 7 retries or after 10 seconds whichever is earlier. What's the initial backoff period?